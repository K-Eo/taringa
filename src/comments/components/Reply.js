import React, { PureComponent } from 'react'

import Commentable from './Commentable'

class Reply extends PureComponent {
  render() {
    return <Commentable {...this.props} />
  }
}

export default Reply
