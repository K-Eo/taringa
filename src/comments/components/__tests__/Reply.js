import React from 'react'

import Reply from '../Reply'

describe('Reply', () => {
  it('exists', () => {
    expect(Reply).toBeDefined()
  })

  it('renders default', () => {
    const wrapper = shallow(<Reply />)

    expect(wrapper).toMatchSnapshot()
  })

  it('passes props', () => {
    const wrapper = shallow(<Reply foo="bar" />)

    expect(wrapper).toMatchSnapshot()
  })
})
