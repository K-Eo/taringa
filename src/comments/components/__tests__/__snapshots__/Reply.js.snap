// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reply passes props 1`] = `
<Commentable
  comment={
    Object {
      "body": "#BODY",
      "create": 0,
      "downvotes": 0,
      "upvotes": 0,
    }
  }
  foo="bar"
  owner={
    Object {
      "avatar": "#AVATAR",
      "username": "#USERNAME",
    }
  }
/>
`;

exports[`Reply renders default 1`] = `
<Commentable
  comment={
    Object {
      "body": "#BODY",
      "create": 0,
      "downvotes": 0,
      "upvotes": 0,
    }
  }
  owner={
    Object {
      "avatar": "#AVATAR",
      "username": "#USERNAME",
    }
  }
/>
`;
